doc/tools/completion-template

```diff
@@ -5,7 +5,11 @@ _FUNCTION_NAME()
     local cur prev split=false
     _get_comp_words_by_ref -n : cur prev
 
-    _split_longopt && split=true
+    if type _comp__split_longopt &>/dev/null; then
+        _comp__split_longopt && split=true
```

Comment:
See https://github.com/scop/bash-completion/issues/1135#issuecomment-1998483710

Reply:
The comment linked by me above mentions a commit that adds a replacement for `_split_longopt` (and inside that commit many of the example scripts were switched from `_split_longopt` to `_init_completion -s` command) so that seems like a viable replacement (possibly with a `_split_longopt` fallback for ancient Linux distros)